interface/menu.dm

```diff
@@ -76,6 +76,23 @@ GLOBAL_LIST_EMPTY(menulist)
 /datum/verbs/menu/Icon/Size
 	checkbox = CHECKBOX_GROUP
 	default = /datum/verbs/menu/Icon/Size/verb/iconstretchtofit
+/datum/verbs/menu/Icon/Size/Load_checked(client/C)
+	var/atom/verb/verbpath = Get_checked(C)
```

Comment:
don't cast these, they're paths not atoms. (and verb doesn't inherit from atom anyway so thats super weird)